### PR TITLE
docs: add info about demo

### PR DESCRIPTION
**Pull request by Chris53897**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,8 @@ forms are complex! If you find a bug, please open an issue!
 
 Ever have a form field that depends on another?
 
+You can find a [Demo with LiveComponent on Symfony ux](https://ux.symfony.com/demos/live-component/dependent-form-fields).
```
Comment (bocharsky-bw) [inline README.md:8]:
```suggestion
You can find a [Demo with LiveComponent on Symfony UX](https://ux.symfony.com/demos/live-component/dependent-form-fields).
```

**Review by bocharsky-bw — APPROVED** [review]
Good idea 👍

**Comment by bocharsky-bw** [comment]
Could you also fix PHP CS Fixer failures here?

**Comment by Chris53897** [comment]
The PHP CS Fixer failures are not related to this PR.
https://github.com/SymfonyCasts/dynamic-forms/pull/33

**Comment by Chris53897** [comment]
I am not sure if it is `UX` or `ux`.
I will commit your changes.

ux

**Comment by bocharsky-bw** [comment]
That's just a logo, correct form is:
Screenshot 2024-07-30 at 21 45 43

**Comment by bocharsky-bw** [comment]
Thank you!

**Pull request merged**